src/umpire/Umpire.cpp

```diff
@@ -173,6 +173,24 @@ void mark_event(const std::string& event)
       [&](auto& e) { e.name("event").category(event::category::metadata).arg("name", event).tag("replay", "true"); });
 }
 
+std::size_t get_total_memory_allocated()
```

Comment:
```suggestion
std::size_t get_total_bytes_allocated()
```
Is it clearer if method name changed to this?

Reply:
yes, good idea